<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629685795743' stop='1629685795967'>
    <name>verify generate-download-share funtion verify project tab</name>
    <title>verify generate-download-share funtion verify project tab</title>
    <test-cases>
        <test-case start='1629685795743' status='failed' stop='1629685795818'>
            <name>Generate-new-dataset.</name>
            <title>Generate-new-dataset.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63150</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63150
                    at ClientRequest.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (node:events:394:28)
                    at Socket.socketErrorListener (node:_http_client:447:9)
                    at Socket.emit (node:events:394:28)
                    at emitErrorNT (node:internal/streams/destroy:157:8)
                    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
                    at processTicksAndRejections (node:internal/process/task_queues:83:21)
                From: Task: WebDriver.takeScreenshot()
                    at Driver.schedule (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at Driver.takeScreenshot (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
                    at run (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\browser.js:58:33)
                    at ProtractorBrowser.to.&lt;computed> [as takeScreenshot] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\browser.js:66:16)
                    at Spec.jasmine.Spec.addExpectationResult (E:\vmware\Loop\GitHub\e2e\protractor.config.js:147:17)
                    at Spec.onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:548:10)
                    at QueueRunner.onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:515:50)
                    at onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4310:14)
                    at handleError (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4243:11)
                    at process.onerror (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:2371:17)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629685795818' status='failed' stop='1629685795899'>
            <name>Download project.</name>
            <title>Download project.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63150</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63150
                    at ClientRequest.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (node:events:394:28)
                    at Socket.socketErrorListener (node:_http_client:447:9)
                    at Socket.emit (node:events:394:28)
                    at emitErrorNT (node:internal/streams/destroy:157:8)
                    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
                    at processTicksAndRejections (node:internal/process/task_queues:83:21)
                From: Task: WebDriver.takeScreenshot()
                    at Driver.schedule (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at Driver.takeScreenshot (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
                    at run (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\browser.js:58:33)
                    at ProtractorBrowser.to.&lt;computed> [as takeScreenshot] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\browser.js:66:16)
                    at Spec.jasmine.Spec.addExpectationResult (E:\vmware\Loop\GitHub\e2e\protractor.config.js:147:17)
                    at Spec.onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:548:10)
                    at QueueRunner.onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:515:50)
                    at onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4310:14)
                    at handleError (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4243:11)
                    at process.onerror (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:2371:17)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629685795899' status='failed' stop='1629685795964'>
            <name>Share project.</name>
            <title>Share project.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63150</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63150
                    at ClientRequest.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (node:events:394:28)
                    at Socket.socketErrorListener (node:_http_client:447:9)
                    at Socket.emit (node:events:394:28)
                    at emitErrorNT (node:internal/streams/destroy:157:8)
                    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
                    at processTicksAndRejections (node:internal/process/task_queues:83:21)
                From: Task: WebDriver.takeScreenshot()
                    at Driver.schedule (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at Driver.takeScreenshot (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
                    at run (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\browser.js:58:33)
                    at ProtractorBrowser.to.&lt;computed> [as takeScreenshot] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\browser.js:66:16)
                    at Spec.jasmine.Spec.addExpectationResult (E:\vmware\Loop\GitHub\e2e\protractor.config.js:147:17)
                    at Spec.onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:548:10)
                    at QueueRunner.onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:515:50)
                    at onException (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4310:14)
                    at handleError (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4243:11)
                    at process.onerror (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:2371:17)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>